routes/experiments.js

```diff
@@ -1,10 +1,15 @@
 const express = require("express");
 const ExperimentDao = require("../data/ExperimentDao");
 const ApiError = require("../model/ApiError");
+const { returnData } = require("./helperMethods.js");
 
 const router = express.Router();
 const experiments = new ExperimentDao();
 
+router.get("/api/test", (req, res) => {
```

Comment:
Maybe some documentation on what this is for. Or more stuff added in for testing.

Reply:
if this route is to test whether the backend is online, it should just be the root route "/", and the returned message could be something like "Welcome to uCredit backend" or "uCredit backend alive"